scripts/lintcommit.lua

```diff
@@ -0,0 +1,34 @@
+current_branch = vim.trim(vim.fn.system({'git', 'branch', '--show-current'}))
+ancestor_commit = vim.trim(vim.fn.system({'git', 'merge-base', 'master', current_branch}))
+commits_on_current_pr = vim.trim(vim.fn.system({'git', 'rev-list', ancestor_commit .. ".." .. current_branch}))
```

Comment:
good practice to use `local`
```suggestion
local current_branch = vim.trim(vim.fn.system({'git', 'branch', '--show-current'}))
local ancestor_commit = vim.trim(vim.fn.system({'git', 'merge-base', 'master', current_branch}))
local commits_on_current_pr = vim.trim(vim.fn.system({'git', 'rev-list', ancestor_commit .. ".." .. current_branch}))
```